sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 0.3.0-beta.4 (Unreleased)
-
-### Features Added
+## 0.3.0-beta.4 (2022-07-29)
 
 ### Breaking Changes
 
-### Bugs Fixed
+- `ServiceVersion` enum values are now pascal case.
 
 ### Other Changes
 
+- Updated `Azure.Core` to `1.25.0`.
```

Comment:
This is an unusual pattern that we don't typically see.  To me, this implies that you'll include all transitive dependency updates in your changelog going forward.   But, that also opens up the complaint that we're not tracking dependencies of dependencies.

In my opinion, it'd be better to not include this unless we have a specific need/reason for doing so.

Reply:
Sure. I can remove them. They're required in Java land, which is why I added them.

Reply:
Also, these aren't transitive dependencies. These are direct dependencies.

Reply:
@jsquire What's the recommendation on direct dependencies?

Reply:
The same; we do not typically detail version bumps unless there's some special scenario that would cause customers to have to take action because of the change.  (for example, breaking changes in a dependency)

Reply:
Thanks for the clarification. I've removed those notes.